### PR TITLE
Add VS Feedback and VS Code links to bug report template list

### PR DESCRIPTION
**Pull request by Eilon**
Maybe this will help direct VS-specific issues to the most appropriate location.

Do we think this might help? Or worth trying it for a while?

### CODE REVIEW

**Comment by Eilon** [comment]
Added VS Code link to https://github.com/microsoft/vscode-dotnettools

**Review by samhouts — APPROVED** [review]

**Review by mcumming — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -3,3 +3,9 @@ contact_links:
   - name: .NET MAUI Community Support
     url: https://docs.microsoft.com/answers/topics/dotnet-maui.html
     about: Please ask and answer questions here.
+  - name: Bug report or feature request for Visual Studio for Windows and Mac
+    url: https://learn.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio?view=vs-2022
+    about: Use Visual Studio's Report a Problem feature for issues and suggestions with Visual Studio tooling
+  - name: Bug report or feature request for Visual Studio Code (including C# Dev Kit)
+    url: https://github.com/microsoft/vscode-dotnettools
```
Comment (mcumming) [inline .github/ISSUE_TEMPLATE/config.yml:10]: This should probably go directly to the issues page

**Review by jsuarezruiz — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (jfversluis) [reply .github/ISSUE_TEMPLATE/config.yml:10]:
```suggestion
    url: https://github.com/microsoft/vscode-dotnettools/issues
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -3,3 +3,9 @@ contact_links:
   - name: .NET MAUI Community Support
     url: https://docs.microsoft.com/answers/topics/dotnet-maui.html
```
Comment (jfversluis) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
    url: https://learn.microsoft.com/answers/topics/dotnet-maui.html
```

**Review by jfversluis — APPROVED** [review]
2 potential improvements, but minor

LGTM!

**Review by rachelkang — APPROVED** [review]

**Review by jfversluis — APPROVED** [review]

**Review by Redth — APPROVED** [review]

**Pull request merged**